### ISSUE TITLE
Publish to OpenVSX marketplace

### ISSUE THREAD

**Issue by tomer-epstein** [SAP/yeoman-ui#146]
https://projects.eclipse.org/projects/ecd.openvsx

**Comment by tomer-epstein** [comment]
https://open-vsx.org/

**Comment by sushiljain1989** [comment]
Is available on VS Code marketplace?

**Comment by idoprz** [comment]
Yes. https://marketplace.visualstudio.com/items?itemName=SAPOS.yeoman-ui

**Comment by arifunlu** [comment]
But the updates are not constantly reflected in the marketplace. Please up-to-date.

**Comment by tomer-epstein** [comment]
Will be updated with new version this weekend.

**Comment by vbangelov** [comment]
This issue has been inactive for a while, so I'm closing it. Also the marketplace is being updated regularly with the respective version updates.
